### PR TITLE
Add image to home page button.

### PR DESCRIPTION
**Pull request by basaam0**
upload
analysis
analytics

### CODE REVIEW

File: src/main/webapp/js/analytics.js [hunk 1]
```diff
@@ -130,3 +130,11 @@ $(window).resize(function() {
   drawStoreChart(storeData);
   drawCategoryChart(categoryData);
 });
+
+/**
+ * Redirects the user to the home page when the home button is
```
Comment (emilyvera) [inline src/main/webapp/js/analytics.js:135]: Don't think "when the home button is clicked" is necessary

File: src/main/webapp/js/receipt-analysis.js [hunk 2]
```diff
@@ -242,7 +242,7 @@ function formatCurrency(event) {
 }
 
 /**
- * Redirects the user to the home page when the "Return to Home" button is
+ * Redirects the user to the home page when the home button is
```
Comment (emilyvera) [inline src/main/webapp/js/receipt-analysis.js:245]: Also edit comment here

File: src/main/webapp/receipt-analysis.html [hunk 3]
```diff
@@ -23,6 +23,10 @@
 <title>Receipt Analysis</title>
 
 <body onload="load()">
+  <button id="home-button" href="/" class="btn btn-info btn-sm mt-2 ml-2" type="button" onclick="redirectHome()">
```
Comment (bryyeh) [inline src/main/webapp/receipt-analysis.html:26]: There is a redirectHome() in common.js and receipt-analysis.js. They're both loaded for this page. I feel that could cause some confusion, maybe name them differently.

File: src/main/webapp/receipt-analysis.html [hunk 4]
```diff
@@ -23,6 +23,16 @@
 <title>Receipt Analysis</title>
 
 <body onload="load()">
+  <button
+    id="home-button"
+    href="/"
```
Comment (acheng1) [inline src/main/webapp/receipt-analysis.html:28]: Why is there an `href` here but not in upload.html?

Comment (basaam0) [reply src/main/webapp/receipt-analysis.html]: Removed.

**Pull request merged**